spec/index.html

```diff
@@ -1426,7 +1426,12 @@ <h2>Generalized RDF Triples, Graphs, and Datasets</h2>
 
 <section id="privacy" class="appendix informative">
   <h2>Privacy Considerations</h2>
-  <p>TODO</p>
+  <p>RDF is used to express arbitrary application data,
+    which may include the expression of personally identifiable information (PII),
+    or other information which could be considered sensitive.
+    Authors publishing such information are advised to carefully
+    consider the needs and use of publishing such information,
+    particularly when not protected by layers of security.</p>
```

Comment:
```suggestion
    particularly whether authorization measures are needed for access to the data.</p>
```

1. The layer(s) are not necessarily part of the data, and maybe how the request reaches the data.
2. No assumption the data itself is involved in the security (security applied the data items, rather than API security).

Reply:
I'm happy to get other ideas in here. I viewed this text as the "minimum viable" that satisfies the requirement, but I'm sure there is more that could be said.